plenum/server/monitor.py

```diff
@@ -582,18 +619,24 @@ def getAvgLatencyForClient(self, identifier: str, *instId: int) -> float:
         """
         if len(self.clientAvgReqLatencies) == 0:
             return 0
-        return self.mean(
-            [self.clientAvgReqLatencies[i][identifier][1] for i in instId])
+        means = []
+        for i in instId:
+            avg_lat = self.clientAvgReqLatencies[i].get_avg_latency(identifier)
+            if avg_lat:
+                means.append(avg_lat)
```

Comment:
This can be shortened to something like
```
means = (lat.get_avg_latency(identifier) for lat in self.clientAvgReqLatencies)
means = [avg for avg in means if avg]
```